lib/Transforms/Scalar/PromoteMalloc.cc

```diff
@@ -53,9 +53,10 @@ class PromoteMalloc : public FunctionPass {
         }
 
         if (!nv)
-          nv = new AllocaInst(v->getType()->getPointerElementType(),
-                              addrSpace, CS.getArgument(0), "malloc", &I);
-
+          nv = new AllocaInst(v->getType()->getPointerElementType(), addrSpace,
+                              CS.getArgument(0), "malloc", &I);
+        IRBuilder<> Builder(F.getContext());
```

Comment:
this looks like dead code...

Reply:
removed change... this was for debugging.